delocate/tools.py

```diff
@@ -382,8 +382,8 @@ def dir2zip(in_dir, zip_fname):
     z.close()
 
 
-def find_package_dirs(root_path):
-    """ Find python package directories in directory `root_path`
+def find_packages(root_path):
```

Comment:
Sorry - I'm a little sensitive about changing the API without warning.

Would you mind making and deprecating something like:

```
def find_package_dirs(root_path):
    return [p for p, is_dir in find_packages(root) if is_dir]
```

and importing into `delocating`?